include/ttmlir/Dialect/TTNN/Utils/OptimizerOverrides.h

```diff
@@ -65,10 +68,22 @@ class OptimizerOverridesHandler {
                                TensorMemoryLayout, tt::ttnn::Layout,
                                tt::DataType);
 
+  // Wrapper methods we use to expose the adders to the python bindings
+  std::unordered_map<std::string, InputLayoutOverrideParams>
+  getInputLayoutOverridesPybindWrapper() const;
```

Comment:
What is the reason to create wrappers instead of changing the existing methods?

Reply:
C++ maps bind to Python dict, it's easier to bind and deal with C++ types instead of LLVM. I use wrappers only for bindings, within the MLIR C++ you can use both of them.